### ISSUE TITLE
Extra Space on Desktop Notification

### ISSUE THREAD

**Issue by Ayymoss** [DirtDiglett/Pressure2#70]
#### Description of the issue.
There's a bit of extra padding on the bottom of a notification, it hurts my eyes.

#### Does it also occur on Steam without the skin installed?
No

#### Are you in the Steam BETA? If so, does the issue occur when opted out of the BETA?
No

#### What Operating System are you using? (Windows 10, Mac OSX 11.1, Linux, Ubuntu etc.)
Windows 10

#### Are you using the latest stable release or using the latest commit version?
Yes

#### A screenshot of the issue.
http://i.imgur.com/am4TKpl.png - Actual Image
http://i.imgur.com/NxsguG7.png - What I mean if you don't quite understand.

#### Steps to reproduce the issue.
Get a notification on Windows.

**Comment by Js41637** [comment]
I'm sorry but how does it hurt your eyes to have a tiny bit of extra space on the notifications?

**Comment by avsaponchik** [comment]
@Js41637 totally unseeable

**Comment by DirtDiglett** [comment]
> it hurts my eyes.

![WAT](http://i0.kym-cdn.com/photos/images/newsfeed/000/173/576/Wat8.jpg?1315930535)

**Comment by FlamingFox911** [comment]
Sarcasm aside, this is a very minor issue, even if it does trigger a certain level of OCD. It could be fixed, but it would require alot of time and effort to change every variable in every resource file that mentions the word "Notification" (of which there are ~25+) to the approriate values. This excludes the time it takes to test the changes and, if need be, modify the images used in these notifications. Some of us have work and studies to do as well.

**TL;DR:** Unless someone is really bored and decides to do this for a while, I doubt this will be the first thing that gets done. The ratio of improvement to time-consumed is very, very low.

If you do feel adventurous, however, you can try to resolve this issue by modifying the aforementioned files locally. If it looks better, submit a pull request to the master or development branches. This is open source so any voluntary help is welcome, pending review of course. Good luck.

**TIP:** If on Windows, use Explorer and search for "System.Filename:~=notif -bak" to find these files. 😄

**Comment by DirtDiglett** [comment]
It's not even an issue, it's intentional as it follows Material Design guidelines for the spacing and sizing pretty much exactly.
